shimmingtoolbox/shim/sequencer.py

```diff
@@ -181,7 +183,12 @@ def _eval_static_shim(opt: Optimizer, nii_fieldmap_orig, nii_mask, coef, slices,
         nib.save(nii_merged_coils, os.path.join(path_output, "merged_coils.nii.gz"))
     unshimmed = nii_fieldmap_orig.get_fdata()
     # If the fieldmap was changed (i.e. only 1 slice) we want to evaluate the output on the original fieldmap
-    merged_coils, _ = opt.merge_coils(unshimmed, nii_fieldmap_orig.affine)
+    if equal_fieldmap:
+        # We extend to make sure that merged_coils @ coef[i_shim] has the same dimensions of corrections[...,i_shim]
+        merged_coils = np.expand_dims(opt.merged_coils[:, :, 1, :], 2)
```

Comment:
What does this line do? Why are we not just using `opt.merged_coils`

Reply:
I fixed this in the last commit !